python/cuml/test/test_pca.py

```diff
@@ -201,7 +201,7 @@ def test_pca_inverse_transform(datatype, input_type,
 
 
 @pytest.mark.parametrize('nrows', [4000, 8000])
-@pytest.mark.parametrize('ncols', [5000, 10000])
+@pytest.mark.parametrize('ncols', [5000, stress_param(20000)])
```

Comment:
Should we just push this in separately without waiting for the MR addition?

Reply:
Removing everything except the change to the test parameters into a new PR.